src/main/java/org/openstreetmap/atlas/geography/atlas/items/LocationItem.java

```diff
@@ -25,6 +28,17 @@ protected LocationItem(final Atlas atlas)
         super(atlas);
     }
 
+    @Override
+    public JsonObject asGeoJsonFeature()
```

Comment:
Same type of comment here: Have a local `asGeojsonFeatureWithoutConnectivity` or something similar be protected, and that will force the sub-classes to use it when they implement the proper `asGeojsonFeature` that is mandated by `AtlasEntity`.

Reply:
Revamped that stuff, should be good